oap-server/microbench/src/main/java/org/apache/skywalking/oap/server/microbench/library/datacarrier/SampleData.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.oap.server.microbench.library.datacarrier;
+
+public class SampleData {
+    private int intValue;
+
+    private String name;
+
+    public int getIntValue() {
+        return intValue;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public SampleData setIntValue(int intValue) {
+        this.intValue = intValue;
+        return this;
+    }
+
+    public SampleData setName(String name) {
+        this.name = name;
+        return this;
+    }
```

Comment:
These methods seem to be useless, let's remove these or if they are needed, let's use lombok `@Getter` / `@Setter` annotation to eliminate the boilerplate codes

Reply:
Yes, I have deleted its get&set. In fact, its parameters are not used, let's keep its parameters for now.